validator/client/beacon-api/genesis.go

```diff
@@ -14,25 +14,27 @@ import (
 )
 
 type GenesisProvider interface {
-	GetGenesis(ctx context.Context) (*beacon.Genesis, *httputil.DefaultJsonError, error)
+	GetGenesis(ctx context.Context) (*beacon.Genesis, error)
```

Comment:
No need to keep returning `DefaultErrorJson`, this type should be returned only from `JsonRestHandler`